android/app/src/main/java/app/organicmaps/maplayer/MapButtonsController.java

```diff
@@ -93,9 +95,29 @@ else if (layoutMode == LayoutMode.planning)
 
     final View zoomFrame = mFrame.findViewById(R.id.zoom_buttons_container);
     mFrame.findViewById(R.id.nav_zoom_in)
-          .setOnClickListener((v) -> mMapButtonClickListener.onMapButtonClick(MapButtons.zoomIn));
+          .setOnClickListener((v) -> {
+            mMapButtonClickListener.onMapButtonClick(MapButtons.zoomIn);
+            mFrame.findViewById(R.id.nav_zoom_out).setEnabled(true);
+            mFrame.findViewById(R.id.nav_zoom_out).setAlpha(1);
+            if(Framework.nativeGetDrawScale() == 20) {
```

Comment:
It would be better to have 20 and 2 constants defined somewhere... Do we have them in C++ code?